console/v1/types_console_cli_download.go

```diff
@@ -0,0 +1,39 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Extension for configuring openshift web console command line interface (CLI) downloads.
+type ConsoleCLIDownload struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	// +required
+	Spec ConsoleCLIDownloadSpec `json:"consoleLinksSpec"`
```

Comment:
JSON name doesn't seem to match (consoleLinks?)

It's not clear to me why we need spec and status for this CRD.

Reply:
Spec for consistency, I agree we should not need status.

Reply:
Sorry, copy/paste err on the json.

Reply:
I suppose we don't need spec either, it's not required.  Initially, I though codegen might complain if we deviated from the typical api object format, but it appears to not matter so long as we have meta & correct build tags.